### DISCUSSION TITLE
Is there any way to paste multiple options into the input and have them be selected?

### DISCUSSION THREAD

**Discussion by JacobBumgarner** [sa-si-dev/virtual-select#232] [Q&A]
E.g., see the my input paste solution for `selectize.js`:
https://github.com/rstudio/shiny/issues/1663#issuecomment-1440934736

The options may be comma separated, space separated, etc. 

Thanks in advance!

**Comment by sa-si-dev** [comment]
@JacobBumgarner This feature not available in the plugin as of now. I will implement it in the future release.
